census/keras/requirements.txt

```diff
@@ -1,3 +1,4 @@
+tensorflow==1.3.0
```

Comment:
Should this be 1.2.0? Otherwise, no guarantees of compatibility with CMLE

Reply:
Good point. Fixed